Makefile

```diff
@@ -43,6 +43,11 @@ deploy: manifests
 	kubectl apply -f config/crd/bases
 	kustomize build config/default | kubectl apply -f -
 
+# Deploy end-to-end controller in the configured Kubernetes cluster in ~/.kube/config
```

Comment:
wording is not perfect here. I'd point out that this deployment is used for end-to-end test

Reply:
Fixed